python/pyspark/ml/classification.py

```diff
@@ -231,6 +232,210 @@ def intercept(self):
         """
         return self._call_java("intercept")
 
+    @property
+    @since("2.0.0")
+    def summary(self):
+        """
+        Gets summary (e.g. residuals, mse, r-squared ) of model on
+        training set. An exception is thrown if
+        `trainingSummary == None`.
+        """
+        java_blrt_summary = self._call_java("summary")
+        return BinaryLogisticRegressionTrainingSummary(java_blrt_summary)
+
+    @property
+    @since("2.0.0")
+    def hasSummary(self):
+        """
+        Indicates whether a training summary exists for this model
+        instance.
+        """
+        return self._call_java("hasSummary")
+
+    """
```

Comment:
Sure I can remove this if that's preferable.  I just didn't want to ignore it without knowing the status of the Scala API.  @mengxr @jkbradley is `evaluate` going to public soon or is it best to not include this at all?